windows/README.md

```diff
@@ -0,0 +1,25 @@
+# Windows Demos
+
+## Table of Contents
+- [Windows Demos](#windows-demos)
+  - [Table of Contents](#table-of-contents)
+  - [About These Demos](#about-these-demos)
+    - [Jobs](#jobs)
+  - [Suggested Usage](#suggested-usage)
+
+## About These Demos
+This category of demos shows examples of Windows Server operations and management with Ansible Automation Platform. The list of demos can be found below. See the [Suggested Usage](#suggested-usage) section of this document for recommendations on how to best use these demos.
+
+### Jobs
+
+- [**WINDOWS / Install IIS**](install_iis.yml) - Install IIS feature with a configurable index.html
+- [**WINDOWS / Patching**](patching.yml) - Apply Windows updates by category and create report
+- [**WINDOWS / Chocolatey install multiple**](windows_choco_multiple.yml) - Install multiple packages using Chocolatey and check versions
+- [**WINDOWS / Chocolatey install specific**](windows_choco_specific.yml) - Install a single given package using Chocolatey
+- [**WINDOWS / Arbitrary Powershell**](arbitrary_powershell.yml) - Run given Powershell script (default: retrieve cat fact from API)
+- [**WINDOWS / Powershell Script**](powershell_script.yml) - Run a Powershell script stored in source control to query services
+- [**WINDOWS / Powershell DSC configuring password requirements**](powershell_dsc.yml) - Configure password complexity with Powershell desired state config
+- [**WINDOWS / Create Active Directory Domain**](active_directory/create_ad_domain.yml) - Create a new AD Domain
```

Comment:
Let's add a note that any AD related jobs are dependent on this one and it is advised to run it prior to a customer call due to taking a few minutes.

Reply:
I'll add to the suggested usage section